frame/balances/src/lib.rs

```diff
@@ -663,6 +672,12 @@ impl<T: Trait<I>, I: Instance> Module<T, I> {
 
 	/// Update the account entry for `who`, given the locks.
 	fn update_locks(who: &T::AccountId, locks: &[BalanceLock<T::Balance>]) {
+		if locks.len() as u32 > T::MaxLocks::get() {
+			frame_support::print(
```

Comment:
This will go into log target `runtime` and `debug` level I think? Better to make it warn. I think you can use `frame_support::debug::warn!(...)`.

Reply:
```suggestion
			frame_support::debug::warn!(
```